wesnoth-thot.po

```diff
@@ -740,26 +698,32 @@ msgid ""
 "continuously as our friend Movrur does. But we have no time to discuss this "
 "further right now; I see trouble brewing down the road!"
 msgstr ""
+"A z praktického hlediska chrání obličej před odletujícími jiskrami a "
+"rozžhavenou ocelí. Dokonce i kovářští mistři z Knalgy si masky navlékají. I "
+"když je asi nenosí tak nepřetržitě jako náš přítel Movrur. Ale teď nemáme "
+"čas o tom dál diskutovat. Vidím, že na cestě vří nějaký problém!"
```

Comment:
```suggestion
"čas o tom dál diskutovat. Vidím, že se na cestě schyluje k nějakému problému!"
```

Reply:
V originále je brewing, což by se dalo pochopi ve smyslu, že se něco peče. Chtěl jsem zachovat ten "vařící" charakter formulace.

Reply:
Možná tedy probublává?

Reply:
já našel ve [slovníku Seznamu](https://slovnik.seznam.cz/preklad/anglicky_cesky/brew) 3. význam chystat se, schylovat se a vybral jsem si to slovo které mi k těm okolním víc sedne (možná trochu ve smyslu něco se peče)

pokud by kontext naznačoval, že na cestě už se bojuje, tak bych si dovolil navrhnout něco jako
na cestě to vře
na cestě vypukla vřava

hledal jsem časování vřít a [našel](https://prirucka.ujc.cas.cz/?id=590) vzory třít, dřít, přít se, takže mi to vaše vří vříská do očí a i s ohledem na to, co už jste přeložil a jak, mi vůbec nesedí

Reply:
No aha, protože to má být vře.
A je pravda, že jsem byl trochu bezradný s tím down. Nevěděl jsem, jestli je se to váže s tím brewing nebo zda je to dole na cestě (což by ale bylo asi z hlediska označení polohy na herní mapě trochu problematické). A tak...
Aj, je to nějak celkově pošahané, kloním se k navrhovanému...